modules/ztp-configuring-pgt-image-registry.adoc

```diff
@@ -43,59 +49,123 @@ sourceFiles:
       storageClassName: image-registry-sc
       volumeMode: Filesystem
   # persistent volume
-  - fileName: ImageRegistryPV.yaml <3>
+  - fileName: ImageRegistryPV.yaml <2>
     policyName: "pv-for-image-registry"
     metadata:
       annotations:
-        ran.openshift.io/ztp-deploy-wave: "100"  <4>
+        ran.openshift.io/ztp-deploy-wave: "100"
   # image registry config
-  - fileName: ImageRegistryConfig.yaml <5>
+  - fileName: ImageRegistryConfig.yaml
     policyName: "config-for-image-registry"
-    complianceType: musthave <5>
+    complianceType: musthave <3>
```

Comment:
This line can be removed as PGT will set it as `musthave` by default. 

Instead just maybe keep <3> as word of warning -- e.g something like "warning: complianceType: mustonlyhave for ImageRegistryConfig.yaml may cause issues"?

Reply:
I think we need to be more explicit than "may cause issues". I've suggested a wording. LMK if it works.